diagnostic.md

```diff
@@ -73,13 +88,26 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request POST "http://localhost:3000/donuts/" \
+--header: "Content-Type: application/json" \
+--data $'{
+    "donut": {
+         "name": "French Cruller",
+       "price": "$0.99"
+    }
+  }'
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function (form) {
+  return $.ajax({
+    url: "http://localhost:3000" + "/donuts",
+    method: 'POST',
+    data: getFormFields(form),
```

Comment:
Since you walked in late you missed this but the data piece here should have been the object itself. But yes:

```js
data: {
   donut:  {
       "name": "French Cruller",
       "price": "$0.99"
    }
};
```